peachjam/adapters/adapters.py

```diff
@@ -345,6 +346,36 @@ def update_document(self, url):
 
         self.set_parent(document, created_doc)
         self.fetch_relationships(document, created_doc)
+        self.download_and_save_document_images(document, created_doc)
+
+    def list_images_from_content_api(self, document):
+        links = document["links"]
+        if links:
+            for link in links:
+                if link["href"].endswith("media.json"):
+                    response = self.client_get(link["href"])
+                    if response.status_code == 200 and response.json()["results"]:
```

Comment:
`self.client_get` will raise an exception on a non-200 response, so you can remove that check.